src/main/java/Storage.java

```diff
@@ -0,0 +1,77 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with loading tasks from the file
+ */
+public class Storage {
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Returns file with the specified file path
+     *
+     * @return file if exist in computer, otherwise create new file
+     */
+    public File getFile() throws IOException {
+        File file = new File(this.filePath);
+        //if file does not exist, create new file and directory
+        if (!file.exists()) {
+            file.createNewFile();
+            file.mkdirs();
+        }
+
+        return file;
+    }
+
+    /**
+     * Returns the path of the file
+     *
+     * @return the path of the file
+     */
+    public String getFilePath() {
+        return this.filePath;
+    }
+
+    /**
+     * Returns an ArrayList of tasks recorded in the hard disk
+     *
+     * @return an ArrayList of tasks that are in the form of
+     *                 event, deadline and todo
+     */
+    public ArrayList<Task> load() throws HenryException {
+        // create a Scanner using the File as the source
+        try{
```

Comment:
Shouldn't there be a space before the opening bracket?
```suggestion
        try {
```